_posts/2020-11-31-homebrew-2.6.0.md

```diff
@@ -0,0 +1,36 @@
+
+Major changes and deprecations since 2.5.0:
+
+- [macOS Big Sur is supported (and High Sierra unsupported)](https://github.com/Homebrew/brew/pull/9054)
+- [`depends_on :java`, `brew switch`, `brew diy` and various other APIs have been deprecated](https://github.com/Homebrew/brew/pull/9209)
+- [All `brew cask` commands have been deprecated in favour of `brew` commands (with `--cask`) when necessary](https://github.com/Homebrew/brew/pull/8899)
+- [`Homebrew.args` is deprecated](https://github.com/Homebrew/brew/pull/8795)
+- [All `Requirement`s are deprecated in Homebrew/core](https://github.com/Homebrew/brew/pull/9052)
+- [macOS Homebrew on M1/Apple Silicon/ARM has partial functionality. We recommend installing into `/opt/homebrew` and forbid installing into `/usr/local` (to avoid clashing with the macOS Intel install and allow their usage side-by-side).](https://github.com/Homebrew/brew/pull/9117)
+- [`brew tap-new` will setup GitHub Actions workflows to](https://github.com/Homebrew/brew/pull/8762) [upload to GitHub Releases](https://github.com/Homebrew/brew/pull/8410). Read [the blog post](https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/) for more documentation.
+- [GitHub deprecated their API's basic authentication](https://github.com/Homebrew/brew/pull/8826)
+
+Other changes since 2.5.0 I'd like to highlight are the following:
+
+- [`brew doctor` checks all tap branches, not just `homebrew-core`](https://github.com/Homebrew/brew/pull/9305)
+- [`brew unbottled` is a new developer command to identify formulae that haven't had binary packages built yet](https://github.com/Homebrew/brew/pull/9308)
+- [`brew install ./` is now recommended for installing local file formulae](https://github.com/Homebrew/brew/pull/9241)
+- [Bash, ZSH and Fish formula completion is now much faster with the `brew formulae` command](https://github.com/Homebrew/brew/pull/8846)
+- [`brew install --force-bottle` refuses to ever build from source](https://github.com/Homebrew/brew/pull/9064)
+- [`brew install` or `brew link` of a versioned keg-only formula will automatically unlink conflicted version formulae](https://github.com/Homebrew/brew/pull/9009)
+- [`brew shellenv` is significantly faster](https://github.com/Homebrew/brew/pull/8904)
+- [Homebrew has further reduced shallow clone usage (on request from GitHub)](https://github.com/Homebrew/brew/pull/8883)
+- [`brew linkage` and commands using the linkage cache have significantly better performance](https://github.com/Homebrew/brew/pull/8695)
+- [`brew bump-cask-pr` is a new developer command to create Homebrew/homebrew-cask pull requests](https://github.com/Homebrew/brew/pull/7986)
+
+Finally:
+
+- [Discourse](https://github.com/Homebrew/brew/pull/9144) and [IRC](https://github.com/Homebrew/brew/pull/8981) are now deprecated as official communication methods in favour of [GitHub Discussions.](https://github.com/Homebrew/brew/pull/8802).
```

Comment:
Is this posted to discourse yet? I seem to recall the last message was "we're trying this other thing"

Reply:
Not posted but seems we’ve agreed on a direction. I’ll post an announcement tomorrow when I post this?